tests/models/qwen2_vl/test_modeling_qwen2_vl.py

```diff
@@ -315,6 +318,42 @@ def test_beam_search_low_memory(self):
     def test_generate_from_inputs_embeds_with_static_cache(self):
         pass
 
+    @unittest.skip("LM test not for VLM")
+    def test_attention_outputs(self):
+        pass
```

Comment:
did all the tests start failing after SequenceClassification was added? These tests should be okay with VLMs so I don't think it is a good idea to skip them. We should rather try to fix it

Reply:
Looks like it is a problem with the pooling of logits that was copied from llamaForSequenceClassification. Will investigate further.